e2e/test/transactions.js

```diff
@@ -27,14 +28,34 @@ describe('transactions', () => {
     assert(toBN(balance).isZero(), 'Account should not have tokens yet')
 
     // send transaction to home chain
-    await homeWeb3.eth.sendTransaction({
+    const depositTx = await homeWeb3.eth.sendTransaction({
       from: user.address,
       to: HOME_BRIDGE_ADDRESS,
       gasPrice: '1',
-      gasLimit: '50000',
+      gas: '50000',
       value: '1000000000000000000'
     })
 
+    // Send a Tx to generate a new block
+    await generateNewBlockWithTx(homeWeb3, user.address)
+
+    // wait to send a Tx to generate a new block
```

Comment:
What we are doing here is a little tricky, so a more verbose comment would help.

We send a transaction that will generate a Deposit event, and then a trivial transaction to generate a new block (since the watcher is configured to wait 1 block). After this, the bridge should create a new transaction with a CollectedSignatures event and, for the same reason, we generate another trivial transaction so that the second watcher gets this.